next.config.js

```diff
@@ -1,5 +1,16 @@
 const { RetryChunkLoadPlugin } = require("webpack-retry-chunk-load-plugin");
 
+const self = "'self' https://*.replay.io wss://*.replay.io";
+const csp = `
+  frame-ancestors ${self};
+  default-src ${self};
+  style-src ${self} 'unsafe-inline';
```

Comment:
I think it'd be worth some comments about why we need `unsafe-inline` and `unsafe-eval`.